includes/definitions/discovery/primekey.yaml

```diff
@@ -0,0 +1,270 @@
+mib: PRIMEKEY-APPLIANCE-MIB
+modules:
+    os:
+        version: 'PRIMEKEY-APPLIANCE-MIB::pkAVersion'
+        serial: 'PRIMEKEY-APPLIANCE-MIB::pkAHsmSerialNumber'
+         # The Appliance S/N is not available via SNMP,
+         # but we can retrieve the S/N for the Hardware Security Module. 
+    sensors:
+        count:
+            options:
+                skip_value_lt: 0
+            data:
+                -
+                    oid: PRIMEKEY-APPLIANCE-MIB::pkASfpLoad1m
+                    num_oid: '.1.3.6.1.4.1.22408.1.1.2.1.5.108.111.97.100.50.1'
+                    descr: ' 1m CPU load'
+                    group: 'CPU'
+                    index: 1
```

Comment:
Hardcoding index like this only allows one sensor for each oid, but perhaps that is intended as I see you hard coded num_oid above too? 
Additionally, index can be a name.  The index field is kind of mislabled.  It should be called something like tag or uid.

Reply:
So oid and num_oid can potentially have the same value? Let me respond to the next question before you answer this.

Reply:
setting num_oid to numeric index followed by {{.$index }} works and avoids the need to change test cases

Reply:
There is only ever one sensor for each oid, it seems PrimeKey don't use mib tables. 

I've tried to set the indices so that things order nicely, e.g. based on the order of the oids in the mib.

Reply:
Just found this:
https://github.com/librenms/librenms/pull/13861
Should I raise another PR to try and align my static indexes?